pandas/io/formats/style.py

```diff
@@ -271,6 +271,85 @@ def __init__(
             thousands=thousands,
         )
 
+    def concat(self, other: Styler) -> Styler:
```

Comment:
is there a case to select a *list* of Stylers? (or better just to chain these)

Reply:
i think better to chain.

might be easy to extend later if necessary but probably more maintenance. Follow-ons to this are make this compatible with to_latex and to_excel so dont want to introduce complexity too early.

Reply:
+1 to chaining

Reply:
ok let's see how this goes, the issue is that this api is now not like other e.g. its a method rather than a function, but seems ok for now.